tasks/gitlab_helpers.py

```diff
@@ -281,3 +283,57 @@ def print_entry_points(ctx):
     print(len(entry_points), 'entry points:')
     for entry_point, config in entry_points.items():
         print(f'- {color_message(entry_point, Color.BOLD)} ({len(config)} components)')
+
+
+def compute_gitlab_ci_config_diff(ctx, before: str, after: str):
+    """
+    Computes the full configs and the diff between two git references.
+    The base commit where after is compared to is the LCA commit between before and after.
+    """
+
+    before_name = before or "merge base"
+    after_name = after or "local files"
```

Comment:
```suggestion
    after_name = after or "HEAD"
```
I suppose. We could also use [this](https://github.com/DataDog/datadog-agent/blob/main/tasks/libs/common/git.py#L53)

Reply:
By default, this is not always `HEAD`. If you test locally and you have uncommitted changes, then local files will be parsed and not files from `HEAD`